components/validator/challenger.go

```diff
@@ -199,6 +199,7 @@ func (c *Challenger) Stop() error {
 
 	c.cancel()
 	c.wg.Wait()
+
```

Comment:
Why do you put blank here? is it for readability?

Reply:
To unify with other modules' `Stop()`.